server/routes/overview/overviewView.ts

```diff
@@ -0,0 +1,79 @@
+import type { FunctionalSkills, InPrisonCourse, InPrisonCourseRecords, PrisonerSummary } from 'viewModels'
+import dateComparator from '../dateComparator'
+
+type RenderArgs = {
+  prisonNumber: string
+  tab: string
+  prisonerSummary: PrisonerSummary
+  functionalSkills: FunctionalSkills
+  inPrisonCourses: InPrisonCourseRecords
+  isPostInduction: boolean
+  lastUpdatedBy: string | null
+  lastUpdatedDate: Date | null
+  lastUpdatedAtPrisonName: string | null
+  noGoals: boolean
+  goalCounts: { activeCount: number; archivedCount: number; completedCount: number }
+  completedCourses: Array<InPrisonCourse>
```

Comment:
Do we need the view to know about the prisonNumber? (not sure any of our other view classes do they? 🤔 ) and also the separation of the different course types (completed, in-progress, withdrawn etc)
Why do we need those? 🤔

Reply:
Removed them.